tests/xtro-sharpie/common-Security.ignore

```diff
@@ -15,3 +15,6 @@
 
 ## untyped enum in Security.framework/Headers/SecPolicy.h but the API use CFOptionFlags
 !unknown-native-enum! SecRevocation bound
+
+## Not bound before, my take is that enums are more descriptive
+!missing-pinvoke! SecCopyErrorMessageString is not bound
```

Comment:
it's actually an interesting one... possibly covering undefined enums and translation 
maybe we could bind it as an extension method on the enum ?

Reply:
> could bind it as an extension method on the enum

Like this idea, will test it.